pkg/rhelv2/rpm/rpm.go

```diff
@@ -57,24 +59,117 @@ const (
 		`[%{FILENAMES}\n]` +
 		`.\n`
 
-	dbPath           = `var/lib/rpm/Packages`
 	contentManifests = `root/buildinfo/content_manifests`
 
 	pkgFmt = `rpmv2`
 )
 
 var contentManifestPattern = regexp.MustCompile(`^root/buildinfo/content_manifests/.*\.json$`)
 
+// rpmDatabaseDir is the directory where the RPM database is expected to be in
+// the container filesystem.
+var rpmDatabaseDir = "var/lib/rpm"
+
+// rpmDatabaseFiles is a slice of all RPM database files for all known backends.
+var rpmDatabaseFiles = []string{
+	// bdb (rpm < 4.16)
+	"Packages",
+	// sqlite (rpm >= 4.16)
+	"rpmdb.sqlite",
+	"rpmdb.sqlite-shm",
```

Comment:
This was the original reason: Tests without those files failed.  I did additional testing using UT and localdev, and the test passed.

I quickly looked into how SQLite works and [the WAL](https://sqlite.org/wal.html) checkpoint is size-based.  It might be possible that when migrating the BDB `testdata/Packages` using `rpmdb`, it only wrote to the `.sqIite-wal`, and me removing it from the filesystem manually left `.sqlite` empty, or partially updated, causing the failed test.  As I kept developing and writing the database over and over, eventually, SQLite checkpointed it?  I suppose this scenario would never happen when scanning images, so I think it's safe to remove it from the Scanner.

Reply:
Let me know if you want to know more about these files, otherwise, we can resolve this conversation.